src/player-config.js

```diff
@@ -78,7 +78,11 @@ const DefaultConfig = {
   drm: {
     keySystem: ''
   },
-  network: {}
+  network: {},
+  visibility: {
+    tolerance: 10,
```

Comment:
i think this name is not to clear, and it will be more correct to get this as an array cause sometimes people will want non linear stops

Reply:
Not relevant anymore